autoware.repos

```diff
@@ -47,3 +47,7 @@ repositories:
     type: git
     url: https://github.com/CPFL/car_demo.git
     version: e364448fad421cb6244c9f828f978d8d877dcbf9
+  vehicles/sd:
```

Comment:
@esteve Should this be `drivers/streetdrone`?

Reply:
@gbiggs not sure, on the one hand yes it's the driver for SD's vehicle interface. However, it's so vehicle-specific and their repository may contain other related packages in the future (e.g. URDF model, simulation worlds, etc.) that I suggested @EfimiaPanagiotaki-StreetDrone to put it in `vehicles/sd`. This is akin to OpenEmbedded, where they have drivers for each component, and then the BSP packages that aggregate the drivers and add configurations.

Reply:
Works for me. :+1: